src/lib/GovernorTimelockCompound.sol

```diff
@@ -0,0 +1,246 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (governance/extensions/GovernorTimelockCompound.sol)
+//
+// The Radworks Timelock is not compatible with the Governor Alpha Timelock interface. This
+// meant we had to fork the contracts that interact with the timelock and replace the Timelock
+// interface with the Radworks Timelock interface. Also, if Radworks decides to change the
+// Timelock in the future then it must conform to the Radworks Timelock interface.
+//
+// Radworks Timelock interface:
+// https://etherscan.io/address/0xB3a87172F555ae2a2AB79Be60B336D2F7D0187f0#code#F1#L306
+// Governor Alpha Timelock interface:
+// https://github.com/compound-finance/compound-protocol/blob/a3214f67b73310d547e00fc578e8355911c9d376/contracts/Governance/GovernorAlpha.sol#L320
+// forgefmt: disable-start
+
+pragma solidity ^0.8.0;
+
+import {Governor} from "@openzeppelin/contracts/governance/Governor.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {IGovernorTimelock} from
+  "@openzeppelin/contracts/governance/extensions/IGovernorTimelock.sol";
+import {IRadworksTimelock} from "src/interfaces/IRadworksTimelock.sol";
+import {Timers} from "@openzeppelin/contracts/utils/Timers.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+import {IERC165} from "@openzeppelin/contracts/interfaces/IERC165.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+
+/**
+ * @dev Extension of {Governor} that binds the execution process to a Compound Timelock. This adds a delay, enforced by
+ * the external timelock to all successful proposal (in addition to the voting duration). The {Governor} needs to be
+ * the admin of the timelock for any operation to be performed. A public, unrestricted,
+ * {GovernorTimelockCompound-__acceptAdmin} is available to accept ownership of the timelock.
+ *
+ * Using this model means the proposal will be operated by the {TimelockController} and not by the {Governor}. Thus,
+ * the assets and permissions must be attached to the {TimelockController}. Any asset sent to the {Governor} will be
+ * inaccessible.
+ *
+ * _Available since v4.3._
+ */
+abstract contract GovernorTimelockCompound is IGovernorTimelock, Governor {
+  using SafeCast for uint256;
+  using Timers for Timers.Timestamp;
+
+  struct ProposalTimelock {
+    Timers.Timestamp timer;
+  }
+
+  /// @dev The interface for this variable was changed to conform to the Radworks Timelock interface.
```

Comment:
nit: feel free to ignore, or if you accept, it could apply elsewhere in the file as well

```suggestion
  /// @dev Modified to conform to the Radworks Timelock interface.
```

Reply:
Done (in 4 places) 4c8b4aa5079ff2ca8a0124eeab3624cc5cfce0db